feelslikemonday/app/src/main/java/com/example/feelslikemonday/ui/followrequest/FollowerRequestFragment.java

```diff
@@ -101,20 +101,20 @@ public void onChanged(@Nullable String s) {
         return root;
     }
 
-    private void storeFollowerPermission(String friendName){
-        //TODO: pass filename variable from MainActivity
-        SharedPreferences mySharedPreferences = this.getActivity().getSharedPreferences("Xiaole", Context.MODE_PRIVATE);
-        SharedPreferences.Editor myEditor = mySharedPreferences.edit();
-        myEditor.putString("followerPermission", friendName);
-        myEditor.apply();
-    }
-
-    private String getFollowerPermission(){
-        SharedPreferences mySharedPrefrences = this.getActivity().getSharedPreferences("Xiaole", Context.MODE_PRIVATE);
-        //String FollowerPermission = mySharedPrefrences.getString("recipientUsername", this.getActivity().);
-        String PermittedName = "Xiaole2";
-        return PermittedName ;
-    }
```

Comment:
why remove this stuff?

Reply:
Its hardcoded stuff not being used right now. If we want it back we can get it in a prior commit